walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -26,26 +26,23 @@
     Collection<String> attendees = request.getAttendees();
     long duration = request.getDuration();
     boolean inattendance = false; 
+    int entireDay = 1440;
```

Comment:
Does this need to be inside this function, or can it be a class var? Also, since it never gets changed, can we make it a final var, and perhaps name it `minutesPerDay`?